internal/librarian/init.go

```diff
@@ -28,6 +28,11 @@ import (
 	"github.com/googleapis/librarian/internal/gitrepo"
 )
 
+const (
+	KeyBreakingChange = "BREAKING CHANGE"
```

Comment:
can you reuse the const in convention_commits? 
https://github.com/googleapis/librarian/blob/7cb24f92fd31578a44aba491049e6c0f8242ad0b/internal/gitrepo/conventional_commits.go#L43

Reply:
Removed this line since it doesn't need anymore.